### DISCUSSION TITLE
Is Next Image are good for parsing image from cdn and used in Mobile Apps

### DISCUSSION THREAD

**Discussion by xxidbr9** [vercel/community#120] [Show & Tell]
I was wondering is that good or bad using 

> _next/image?url=<url>&w=<w>&q=<q>

for compress the image assets, and use that image in mobile apps?

**Comment by Pieparker** [comment]
It's [how the Vercel network caches different size optimised images](https://vercel.com/docs/concepts/next.js/image-optimization#edge-network-optimization) so I see no reason why not!

**Reply by xxidbr9** [reply]
Thanks for the reply, I was used currently on mobile apps and I don't have any problems with that, I'm too lazy to build an image optimizing service for the backend, so that's why I asked first, btw that helps me a lot.

Thank Vercel team 🚀
